members/models/activity.py

```diff
@@ -11,6 +11,10 @@ class Meta:
         verbose_name_plural = "Aktiviteter"
         ordering = ["department__address__zipcode", "start_date"]
 
+    MEMBER_AMOUNT = 75
```

Comment:
```suggestion
    MEMBERSHIP_MIN_AMOUNT = 75
```
? synes jeg er nemmere at forstå 🙂